#request field
uint32 buffer_size
---
#response field
bool result
